Platform/OpenCanopy/Views/BootPicker.c

```diff
@@ -1396,6 +1396,15 @@ BootPickerEntriesSet (
       case OC_BOOT_RESET_NVRAM:
         Status = CopyLabel (&VolumeEntry->Label, &GuiContext->Labels[LABEL_RESET_NVRAM]);
         break;
+      case OC_BOOT_TOGGLE_SIP:
+        if (StrCmp (Entry->Name, OC_MENU_SIP_IS_DISABLED)) {
+          Status = CopyLabel (&VolumeEntry->Label, &GuiContext->Labels[LABEL_SIP_IS_ENABLED]);
+        } else if (StrCmp (Entry->Name, OC_MENU_SIP_IS_ENABLED)) {
+          Status = CopyLabel (&VolumeEntry->Label, &GuiContext->Labels[LABEL_SIP_IS_DISABLED]);
+        } else {
+          ASSERT (FALSE);
```

Comment:
@vit9696 - Still not entirely sure of the logic of these, should I insert e.g. `CpuDeadLoop` after this, or is okay as is?

Reply:
There is an UNREACHABLE macro. You can use it.

Reply:
Okay - not so much worried about compiler - unless it warns - as about what would actually happen - where the instruction point would run to next. Not clear if UNREACHABLE stops the CPU from progressing - would imagine not - and not clear if ASSERT stops the CPU from progressing (e.g. in RELEASE build)? Guess I need to compile and run some tests.

Reply:
Okay - thanks for pms clarifying! Will change as discussed.